torch/_higher_order_ops/triton_kernel_wrap.py

```diff
@@ -931,3 +931,156 @@ def call_triton_kernel(self, variable, args, kwargs, tx):
             grids = [grids[0]]
 
         return self.call_HOP(variable, grids, combined_args_raw, tx)
+
+
+###############################################################################
+# capture_triton API that makes a user-defined triton kernel traceable into
+# a graph via make_fx or non-strict export (coming soon)
+
+
+def capture_triton(triton_kernel, /):
```

Comment:
do we ever expect more args here? i.e. why do we need the `/`?

Reply:
The / makes the `triton_kernel` argument positional-only (so a user cannot do `capture_triton(triton_kernel=blah)`). I like adding it to APIs so that we get freedom to change the names of arguments without being BC-breaking